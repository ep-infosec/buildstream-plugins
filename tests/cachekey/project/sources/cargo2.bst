kind: import
sources:
- kind: local
  path: files/local
- kind: cargo
  url: upstream:crates
  vendor-dir: ponies
  cargo-lock: Pony.lock
  ref:
  - name: foo
    version: 1.0
    sha: thisisthecratesha
