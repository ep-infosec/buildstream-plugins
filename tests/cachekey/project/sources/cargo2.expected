9013b5ea603e56ce4690fdc7a48ccf9d400e00dcd89cfd115be7d55403f74017